### ISSUE TITLE
Does StatDCAT-AP consider the case that a dataset consists of multiple (related) data tables?

### ISSUE THREAD

**Issue by pvgenuchten** [SEMICeu/StatDCAT-AP#4]
This is more a question than an issue, i was reading the spec and noticed `stat:attribute` seems a direct property of `dcat:Dataset`. Does this mean only a single data-table can be described using 1 StatDCAT-AP record?

In geography we use [iso19110](https://www.iso.org/standard/39965.html) to describe data models, in that domain a dataset can be described using multiple feature-classes (tables). But maybe this approach is less common in statistics?

**Comment by jakubklimek** [comment]
In DCAT-AP, all distributions of a single dataset should have similar content. Each distribution can express that content in, e.g., a different format. As StatDCAT-AP is a DCAT-AP profile, the same applies to it.

If you have multiple files, each with a different content but related, you can describe each one as a separate Dataset with a single Distribution, and then group them using [Dataset Series](https://semiceu.github.io/StatDCAT-AP/drafts/3.0.0/#DatasetSeries), which is currently being considered for StatDCAT-AP 3.0.0.

**Comment by pvgenuchten** [comment]
I was thinking about a table of trees and a table of leaves referencing trees, I guess it would be considered as 2 (related) datasets

**Comment by jakubklimek** [comment]
Yes. Alternatively, you could package the files and indicate so using [Distribution.packagingFormat](https://semiceu.github.io/StatDCAT-AP/drafts/3.0.0/#Distribution.packagingformat), but that prohibits you from describing the packaged files individually, e.g., their schema, etc.
